src/components/home-components/search-bar/SearchInput.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useMemo } from 'react';
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+import { autocompleteCompanyListAtoms } from '../../../recoil/atoms';
+import styled from '@emotion/styled';
+import { AutoComplete as AntdAutoComplete, AutoCompleteProps as AntdAutoCompleteProps } from 'antd';
+import search from '../../../axios';
+import { AutoCompleteCompanyList } from '../../../types';
+import { getRegExp } from 'korean-regexp';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const AutoComplete = styled((props: AntdAutoCompleteProps) => <AntdAutoComplete {...props} />)`
+  text-align: center;
+  justify-self: center;
+  font-size: 2rem;
+  width: 70%;
+  max-width: 500px;
+`;
+
+type Props = {
+  searchText: string;
+  setSearchText: (company: string) => void;
+};
+
+function SearchInput({ searchText, setSearchText }: Props) {
+  const autoCompleteCompanyList = useRecoilValue(autocompleteCompanyListAtoms);
+  const setAutoCompleteCompanyList = useSetRecoilState(autocompleteCompanyListAtoms);
+
+  const options = useMemo(() => {
+    if (!searchText || !autoCompleteCompanyList || !autoCompleteCompanyList?.companies || !autoCompleteCompanyList.companies?.length) {
+      return;
+    }
+
+    const { companies } = autoCompleteCompanyList;
+
+    return companies.map((companyName) => ({ label: companyName, value: companyName }));
+  }, [searchText, autoCompleteCompanyList]);
+
+  useEffect(() => {
+    if (!searchText) {
+      return;
+    }
```

Comment:
```suggestion
    if (!searchText) return;
```

개인적으로 한 줄 조건문에 중괄호 굉장히 시러해서... ㅎㅎ

Reply:
아 혹시 prettier나 lint가 이렇게 쓰라고 강제한거면 이 리뷰는 그냥 무시하셔도 됨다
아니라면 다른 한줄짜리에도 적용 부탁드립니다 🙇

Reply:
아녜요! 바로 수정해두겠습니다! 
의견 바로 주셔서 감사합니다 ㅎㅎ